kuma/core/tests/test_commands.py

```diff
@@ -6,7 +6,7 @@
 def test_help():
     with pytest.raises(CommandError) as excinfo:
         call_command('ihavepower', stdout=StringIO())
-    assert str(excinfo.value) == 'Error: too few arguments'
+    assert 'Error: the following arguments are required: username' == str(excinfo.value)
```

Comment:
This change will cause issues in Python 2. I don't see any solutions that works for both environment.